components/knative-function-controller/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/ @rakesh-garimella @sayanh @venturasr @lilitgh @k15r @nachtmaar
```

Comment:
Do you need a separate CODEOWNERS for this component?, i think it should be covered via CODEOWNERS file at root level

Reply:
True we have CODEOWNERS file at root repo.